### ISSUE TITLE
create/edit/delete the sections from group management screen

### ISSUE THREAD

**Issue by petervojtek** [PowerlineApp/powerline-mobile#337]
from #309:

> group divisions is part of frontbundle functionality that needs to be available as API endpoint. Create/delete group division(s), name/rename group division(s), assign/remove users to (potentially multiple) group divisions... This is low priority compared to notifications or anything relating to Peter's questions.

**Comment by jterps08** [comment]
@igaponov please clarify if group divisions are exposed via API

**Comment by igaponov** [comment]
"Group divisions" are called "Group sections" on backend, so please check `.../group-sections` and `.../groups/{id}/sections` endpoints in API docs for more information.

**Comment by jterps08** [comment]
That works... thanks @igaponov

**Comment by petervojtek** [comment]
@jterps08 , it's done.

the design it not the most beautiful, but we can tweak that anytime later.
from functional point of view it has the ability to create/delete sections and to add/remove users from each section.

<img width="330" alt="screen shot 2016-11-10 at 09 50 07" src="https://cloud.githubusercontent.com/assets/225506/20170091/246ae7d8-a72b-11e6-820e-cc27b3fcbdaa.png">

**Comment by jterps08** [comment]
Thanks Peter - yeah, this will work for now, but in the future we'll need a different UX/UI for groups that have hundreds or thousands of users... thank you!

**Comment by petervojtek** [comment]
>  in the future we'll need a different UX/UI for groups that have hundreds or thousands of users

you are right @jterps08, thought that will need also redesign the API because currently it does not allows to mass add/remove users from section, i.e. currently 1 user add/remove = 1 http request.

**Comment by petervojtek** [comment]
I realized I messed the issues, there are two tasks related to group sections:
- create/edit/delete the sections from group management screen -- I have done this inside this issue and now renamed this issue to properly address what I have done.
- pick a section or whole group when creating leader content to determine target users of the content. -- this is not yet done, I created new issue #414 for it.
